tests/spec/testing_spec.lua

```
@@ -0,0 +1,397 @@
+  },
+  json = {
+    decode = function(str)
+      if str:find("jest") then
+        return {devDependencies = {jest = "^29.0.0"}}
+      end
+      return {}
+    end,
+  },
+  o = { columns = 120, lines = 40 },
+  log = { levels = { ERROR = 1, WARN = 2, INFO = 3 } },
+  notify = function(msg, level) 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)